### DISCUSSION TITLE
关于自定义渲染节点

### DISCUSSION THREAD

**Discussion by tiancihe** [antvis/X6#1131] [General]
自定义渲染html/React/Vue节点时，shape的值可以改成业务自定义值而不是'html'/'react-shape'/'vue-shape'吗

比如增加一个配置项叫 renderType: 'svg' | 'html' | 'react' | 'vue'

而不是把渲染类型跟 shape 绑定

**Comment by ghost** [comment]
现在所有的节点，其实都是基于 svg 渲染的……

你提到的 html'/'react-shape'/'vue-shape'，其实是基于 svg 的扩展，渲染容器为 foreignObject 标签 

shape 的值，可以使用自定义组件的形式设定为你需要的业务自定义值。
